tracer.cpp

```diff
@@ -0,0 +1,52 @@
+#ifdef __cplusplus
```

Comment:
Only curious: What's this file good for?

Reply:
gcc and clang have instrumentation options via the `-finstrument-functions` compile option. The file was an attempt to get traces of function calls (and worked also somehow). There is a much better and already ready tool called uftrace.

The file was included by mistake and forcefully removed few hours later...